tools/common.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup>
+    <VersionSuffix>$([System.DateTime]::Now.ToString(eyyMMdd-1))</VersionSuffix>
```

Comment:
Thanks.

So, for example: "0.1.0-e170206-2"
Where does the hyphen come from between the version prefix and version suffix? Or does it automatically concatenate with hyphen in between?
```C#
<VersionPrefix>0.1.0</VersionPrefix> 
<VersionSuffix>$([System.DateTime]::Now.ToString(eyyMMdd-1))</VersionSuffix> 
```